src/windows/sentry_windows_path.c

```diff
@@ -189,7 +189,7 @@ sentry__path_clone(const sentry_path_t *path)
     if (!rv) {
         return NULL;
     }
-    rv->path = wcsdup(path->path);
+    rv->path = _wcsdup(path->path);
```

Comment:
This calls malloc internally and is not "signal" safe, I guess. Are we okay with that?